[doc] Spec NTFS-FR : adding lines and networks (#518)
CanalTP/transit_model: src/documentation/ntfs_to_netex_france_specs.md

- `ParticipantRef` : The ParticipantRef code must be provided,
- `StopProviderCode` : The code used to identify the provider in the stop_id generation.

## Id of objects

> According to the Netex Specification, object identifiers should have a specific formating.

Reviewer: not sure
```suggestion
According to the Netex Specification, object identifiers should have a specific formatting.
```

> The formating for stops is describe in the [arrets.xml](#arretsxml) part of this specification.

Reviewer: ```suggestion
The formatting for stops is described in the [arrets.xml](#arretsxml) part of this specification.
```

> For the other objects, the following format is used: **FR:[object type]:[object id]:**

Reviewer 2: The `object_id` might create problems if it contains `:`. We need to replace all `:` (with `_` I suppose, like for stops' identifiers?).

- in monomodalStopPlace: all stop_points should have the same name. If not, 2 StopPlace must be created,
- multimodalStopPlace: show the higher rated mode value only

## lignes.xml

### Top level structure

Example: 

>   <?xml version="1.0" encoding="UTF-8"?>
    <CompositeFrame 
            id="FR:CompositeFrame:NETEX-20190709T093214Z:LOC" 

>             version="1.8">

Reviewer 2: Probably another version?
```suggestion
            version="any">
```

Reviewer 2: We'll leave it as it is for now. Following what we have in NeTEx IDF at the moment.
